apps/api/src/coscrad-cli/ingest-media-items.cli-command.spec.ts

```diff
@@ -0,0 +1,131 @@
+
+    let testRepositoryProvider: TestRepositoryProvider;
+
+    let databaseProvider: ArangoDatabaseProvider;
+
+    let testAppModule: TestingModule;
+
+    beforeAll(async () => {
+        testAppModule = await Test.createTestingModule({
+            imports: [
+                CommandModule,
+                EventModule,
+                DynamicDataTypeModule,
+                PersistenceModule.forRootAsync(),
+                MediaItemModule,
+            ],
+            providers: [
+                {
+                    provide: ConfigService,
+                    useFactory: () =>
+                        buildMockConfigServiceSpec(
+                            {
+                                ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+                            },
+                            buildConfigFilePath(Environment.test)
+                        ),
+                },
+                {
+                    provide: TestRepositoryProvider,
+                    useFactory: (
+                        databaseProvider: ArangoDatabaseProvider,
+                        coscradEventFactory: CoscradEventFactory,
+                        dynamicDataTypeFinderService: DynamicDataTypeFinderService
+                    ) =>
+                        new TestRepositoryProvider(
+                            databaseProvider,
+                            coscradEventFactory,
+                            dynamicDataTypeFinderService
+                        ),
+                    inject: [
+                        ArangoDatabaseProvider,
+                        CoscradEventFactory,
+                        DynamicDataTypeFinderService,
+                    ],
+                },
+            ],
+        }).compile();
+
+        // createTestModule({
+        //     ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+        // });
+
+        await testAppModule.init();
+
+        const dynamicDataTypeFinderService = testAppModule.get(DynamicDataTypeFinderService);
+
+        await dynamicDataTypeFinderService.bootstrapDynamicTypes();
+
+        const arangoConnectionProvider =
+            testAppModule.get<ArangoConnectionProvider>(ArangoConnectionProvider);
+
+        const _dbName = arangoConnectionProvider.getDatabaseName();
+
+        databaseProvider = new ArangoDatabaseProvider(arangoConnectionProvider);
+
+        testRepositoryProvider = testAppModule.get(TestRepositoryProvider);
+
+        commandInstance = await CommandTestFactory.createTestingCommand({
+            imports: [CoscradCliModule],
+        })
+            .overrideProvider(AppModule)
+            .useValue(testAppModule)
+            .overrideProvider(ArangoDatabaseProvider)
+            .useValue(databaseProvider)
+            .compile();
+    });
+
+    beforeEach(async () => {
+        await testRepositoryProvider.testTeardown();
+    });
+
+    describe(`when the media directory has some allowed media items only`, () => {
```

Comment:
Add TODOs for additional test coverage and explain why it is not that important right now because this is not publicly available behaviour.

Reply:
done